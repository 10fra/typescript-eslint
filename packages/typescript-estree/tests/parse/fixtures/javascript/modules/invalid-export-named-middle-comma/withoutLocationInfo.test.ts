import path from 'path';
import { testWithoutLocation } from 'test-fixture';

testWithoutLocation(
  path.resolve(
    process.cwd(),
    '..',
    'shared-fixtures',
    'fixtures/javascript/modules/invalid-export-named-middle-comma.src.js',
  ),
  {
    useJSXTextNode: false,
  },
);
