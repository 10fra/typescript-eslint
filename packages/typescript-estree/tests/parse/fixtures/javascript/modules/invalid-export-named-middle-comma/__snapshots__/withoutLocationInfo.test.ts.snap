// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/Users/bradzacher/github/typescript-eslint/packages/shared-fixtures/fixtures/javascript/modules/invalid-export-named-middle-comma.src.js parses without location information 1`] = `"Identifier expected."`;
